pkg/jsonlog/jsonlog_marshalling.go

```diff
@@ -81,8 +91,6 @@ func (mj *JSONLog) MarshalJSONBuf(buf *bytes.Buffer) error {
 	if len(mj.Log) != 0 {
 		if first == true {
```

Comment:
Since there's always a "time" element added, why not dispense with all of the 'if first' logic, and do the buf.WriteString(`,`) after the ffjson_WriteJsonString.